### PR TITLE
@eessex => Adds tracking to SeriesAbout description partner link

### PR DESCRIPTION
**Pull request by kanaabe**
Looks like we missed this analytics call when making Series.

### CODE REVIEW

File: src/Components/Publishing/Series/SeriesAbout.tsx [hunk 1]
```diff
@@ -10,67 +11,90 @@ interface Props {
...
 
+  componentDidMount() {
+    const textLink = document.querySelector(".SeriesAbout__description a")
+    if (textLink) {
+      textLink.addEventListener("click", this.onClickFooterLink)
+    }
+  }
+
+  onClickFooterLink = event => {
+    this.props.tracking.trackEvent({
```
Comment (eessex) [inline src/Components/Publishing/Series/SeriesAbout.tsx]: 👍

**Comment by eessex** [comment]
Forgot to mention that we should use the new commit syntax for automatic NPM releases!

**Comment by eessex** [comment]
about to do one more QA pass for the circle config, so this will be included in the next release -- no change needed on your side.

**Comment by kanaabe** [comment]
OLALA!! 🥂

**Pull request merged**